department-of-veterans-affairs/abd-vro
# Feature - update DC Lookup Table mappings for Contention Classifier

## What was the problem?

This PR is part 1 of 2 parts in order to update lookup table functionality to increase coverage.
This PR only covers updates to the diagnostic code mappings.

Associated tickets or Slack threads:
- https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/459

## How does this fix it?[^1]

 - New DC mappings from google sheet [HERE](https://docs.google.com/spreadsheets/d/1GixpuHPmxEP2UZMEA8XmuvspBjziOTU18zwRcqQLzkk/edit#gid=1296129655) are checked in.
 - DC lookup table version incremented

## How to test this PR
- `cd domain-cc/cc-app`
- `pytest`
- Note `test.test_api.pytest_v5_diagnostic_code()` adds coverage for new diagnostic code not included in previous versions of the lookup table 


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
